scenarios/fuel_greeter/index.html

```diff
@@ -0,0 +1,70 @@
+
+<!--
+ *  Copyright (c) 2021 GraphQL Contributors
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the license found in the
+ *  LICENSE file in the root directory of this source tree.
+-->
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>GraphiQL</title>
+    <style>
+      body {
+        height: 100%;
+        margin: 0;
+        width: 100%;
+        overflow: hidden;
+      }
+
+      #graphiql {
```

Comment:
I think this is an artifact from the old fork. Maybe we remove this? It's just graphql playgroud alternative to hasura.